src/greynirseq/utils/preprocessing/filters.py

```diff
@@ -41,30 +85,10 @@ def safe_div(a, b):
     return a / b
 
 
-class Example:
-    def __init__(self, src, tgt, file_id=None, align_score=None):
-        self.source = src
-        self.target = tgt
-        self.file_id = file_id
-        self.align_score = align_score
-
-    def __str__(self):
-        return "\t".join([self.source, self.target])
-
-    def __repr__(self):
-        ret = [self.source, self.target, self.file_id, self.align_score]
-        return "[" + "\t".join(ret) + "]"
-
-
-def print_ex(ex):
-    print("\t".join([ex["en"], ex["is"]]))
-
-
-def get_or_initialize_encoder():
-    global ENC
-    # This used to have subword encoder from t2t for subword edit-distance.
-    # Removed until this is implemented with fairseq/huggingface/spm
-    raise NotImplementedError
+# TODO: Add support for BPE encoding
+def encode(text: str):
+    """BPE encodes a string."""
+    return text
```

Comment:
I think we should implement this. Filtering sentences so that they do not contain too many subwords is useful for training. If a training example is too long the training does not start. BPE encoding is time-consuming but might save time down the line.